pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -261,6 +261,13 @@ func (r *rbac) getNamespacesToBeReconciled(ctx context.Context) ([]corev1.Namesp
 		// We must make sure that the default SCC is in force via the ClusterRole.
 		sccRoleBinding, err := r.kubeClientSet.RbacV1().RoleBindings(ns.Name).Get(ctx, pipelinesSCCRoleBinding, metav1.GetOptions{})
 		if err != nil {
+			// https://github.com/tektoncd/operator/issues/2100
+			// Reconcile a namespace again with missing RoleBinding
+			if errors.IsNotFound(err) {
+				logger.Infof("RoleBinding %s is missing in namespace: %s, will reconcile again the namespace %s", pipelinesSCCRoleBinding, ns.Name, ns.Name)
+				namespaces = append(namespaces, ns)
+				continue
+			}
 			return nil, err
```

Comment:
can we custumize this error a bit also with error fetching rolebinding `name` from `namespace`: err

Reply:
yes done thanks @piyush-garg

Reply:
I think I was not clear, error message in the logger was fine, i am saying about this for the error we are returning to the end

Reply:
If I understand correctly, you asking to print the error message with the logger on line number 271, then return `nil, err`.
Am I correct @piyush-garg ?

Reply:
I think @piyush-garg is asking to wrap the error

Reply:
yes to wrap the error and return, not logger